### PR TITLE
Dev mariateo recommendations finished section

### PR DESCRIPTION
**Pull request by MariaTeo**

### CODE REVIEW

File: src/_component/config/reusable.scss [hunk 1]
```diff
@@ -101,3 +101,24 @@
   font-size: 14px;
   color: #808080;
 }
+
+%box__with-arrow {
```
Comment (QuintyHH) [inline src/_component/config/reusable.scss:135]: What does % do? 🤷‍♂️

**Review by Constantin-Matei — APPROVED** [review]
:) Nice!

**Review by QuintyHH — APPROVED** [review]
Interesting use of placeholder, it should work if used like this, but I'm not entirely sure there's a reason to use it here :)

Good job tho!

File: src/_component/config/reusable.scss (hunk 1 above)
Comment (MariaTeo) [reply src/_component/config/reusable.scss:135]: It was a mistake, i forgot to delete

**Review by biggicode — APPROVED** [review]

**Review by M00N89 — APPROVED** [review]

**Review by psyhedeliq — APPROVED** [review]

**Pull request merged**